Source/WebCore/html/HTMLTableCellElement.h

```diff
@@ -33,8 +33,19 @@ class HTMLTableCellElement final : public HTMLTablePartElement {
     WTF_MAKE_TZONE_OR_ISO_ALLOCATED(HTMLTableCellElement);
     WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR(HTMLTableCellElement);
 public:
+    // These limits are defined in the HTML specification:
+    // - https://html.spec.whatwg.org/#dom-tdth-colspan
+    // - https://html.spec.whatwg.org/#dom-tdth-rowspan
+    static const unsigned minColspan = 1;
+    static const unsigned maxColspan = 1000;
+    static const unsigned defaultColspan = 1;
+    static const unsigned minRowspan = 0;
+    static const unsigned maxRowspan = 65534;
+    static const unsigned defaultRowspan = 1;
+
     static Ref<HTMLTableCellElement> create(const QualifiedName&, Document&);
 
+    static unsigned getMaxRowspan() { return maxRowspan; }
```

Comment:
we don't use `get` prefixes in the WebKit codebase

Just refer to the constant as `HTMLTableCellElement::maxRowSpan` without adding this new method ?

Reply:
removed.